contents/handbook/company/offsites.md

```diff
@@ -87,6 +87,11 @@ Below is a rough timeline for planning your next offsite, as well as links to te
   - To simplify this process, we give all team members access to a company card, and we ask people to book their own flights
   - We strongly recommend this approach as centralizing flight booking can be a huge headache for offsite lead, and this allows team members to accurately enter their personal information including airline frequent flier and trusted traveller numbers
   - Encourage folks to buy flights early and with the option to refund if they are unable to attend to save on costs
+  - In the event that a new team member will be attending an offsite, but has not started yet, please contact the Ops team to help coordinate. In these cases, the process is:
```

Comment:
We'll have a similar issue with in person onboarding occasionally, so might be worth clarifying that as well?

Reply:
I'll add that in as well